packages/mobile/src/account/saga.ts

```diff
@@ -65,8 +65,10 @@ export function* getPincode(useCache = true) {
       }
     }
 
-    const pincodeEntered = new Promise((resolve, reject) => {
-      navigate(Screens.PincodeConfirmation, { resolve, reject })
```

Comment:
Because `PincodeConfirmation` will try again and again to ask for correct PIN there is no need for `reject`

Reply:
How many times will it try? There should be some max